extensions/amp-story/0.1/test/test-amp-story.js

```diff
@@ -49,6 +50,7 @@ describes.realWin('amp-story', {
     return Array(count).fill(undefined).map((unused, i) => {
       const page = win.document.createElement('amp-story-page');
       page.id = opt_ids && opt_ids[i] ? opt_ids[i] : `-page-${i}`;
+      page.getImpl = () => Promise.resolve(new AmpStoryPage(page));
```

Comment:
Is this still necessary with a file-level import (above)?

Reply:
Unfortunately yes, without this line it'd break, even with the file-level import

Reply:
Something's not right, but ok for now if it works.